src/cms-content/vaccines/state-vaccine-phases.json

```diff
@@ -2236,7 +2236,7 @@
           "phase": "n/a",
           "tier": "",
           "currentlyEligible": false,
-          "description": "* Utahns between the ages of 65-69 \n* Anyone 18 or older with certain medical conditions",
+          "description": "\\* Anyone 18 or older with certain medical conditions",
```

Comment:
This is not going to work: `\\*` escapes the `*` so is not a list item anymore, we need to remove the `\\` before merging